services/brig/src/Brig/API/Internal.hs

```diff
@@ -662,6 +673,23 @@ updateRichInfo uid rup = do
   -- Intra.onUserEvent uid (Just conn) (richInfoUpdate uid ri)
   lift $ wrapClient $ Data.updateRichInfo uid (mkRichInfoAssocList richInfo)
 
+updateLocaleH :: UserId ::: JSON ::: JsonRequest LocaleUpdate -> (Handler r) Response
+updateLocaleH (uid ::: _ ::: req) = do
```

Comment:
we're migrating away from `wai-route`, would it be too hard to create these as servant end-points?